app/kuma-dp/pkg/dataplane/meshmetrics/component.go

```diff
@@ -285,6 +296,17 @@ func (cf *ConfigFetcher) reconfigureBackendIfNeeded(backendName string, backend
 	return nil
 }
 
+func (cf *ConfigFetcher) shutDownMetricsExporters() {
+	ctx, cancel := context.WithTimeout(context.Background(), cf.drainTime)
+	defer cancel()
+	for _, exporter := range cf.runningBackends {
+		err := exporter.exporter.Shutdown(ctx)
```

Comment:
I _suppose_ this could be done in parallel and with waitgroup to increase the chances that all are shutdown? But I don't think it's likely to matter in practice.

Reply:
I think that users will very rarely use more than one exporter